README.md

```diff
@@ -1,168 +1,96 @@
-Selenium Tests for addons.mozilla.org (amo)
-====================
+Tests for addons.mozilla.org (amo)
+==================================
 
-Thank you for checking out Mozilla's Addon-Tests test suite. Mozilla and the Mozwebqa team are grateful for the help and hard work of many contributors like yourself.
-The following contributors have submitted pull requests to Addon-Tests:
-
-https://github.com/mozilla/Addon-Tests/contributors
+Thank you for checking out Mozilla's Addon-Tests test suite.
+This repository contains Selenium tests used to test the website addons.mozilla.org.
 
-Continuous Integration
-----------------------
+[![license](https://img.shields.io/badge/license-MPL%202.0-blue.svg)](https://github.com/mozilla/Addon-Tests/blob/master/LICENSE)
+[![travis](https://img.shields.io/travis/mozilla/Addon-Tests.svg?label=travis)](http://travis-ci.org/mozilla/Addon-Tests/)
+[![dev](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/amo.dev.svg?label=dev)](https://webqa-ci.mozilla.com/view/AMO/job/amo.dev/)
```

Comment:
Just a thought, but I think we should drop the view from the URLs in case we rename or remove it in the future. It's not likely, but we can instead point directly at the job.